server/apps/distributed-app/sample-configuration/rabbitmq.properties

```diff
@@ -5,6 +5,10 @@
 # Mandatory
 uri=amqp://rabbitmq:5672
```

Comment:
Should we deprecate `uri` property? 
When we declare `hosts` property, what is `uri` sense?
Should we have only one property for endpoint declaration? IMO

Reply:
> Should we deprecate uri property?

If I can avoid a breaking change I'm happy.

I think URI is still easier to use for a single Rabbit set up.

You can further more pass extra info in it: VHost, username + password, etc...

> When we declare hosts property, what is uri sense?

 1. Host is optional
 2. URI can be used for vhost, username, password, etc...
  
> Should we have only one property for endpoint declaration? IMO

We have to balance this with other goals too: how about avoiding to much breaking changes?

Reply:
Thanks for answer